mpt/impl/imap-mailbox/external-james/src/test/java/org/apache/james/mpt/imapmailbox/external/james/DeploymentValidation.java

```diff
@@ -55,4 +58,14 @@ public void validateDeployment() throws Exception {
         simpleScriptedTestProtocol.run("ValidateDeployment");
     }
 
+
+    @Test
+    public void ValidateDeploymentWithMailsFromSmtp() throws Exception {
+        SMTPMessageSender smtpMessageSender = new SMTPMessageSender("another-domain");
+        smtpSystem.connect(smtpMessageSender).sendMessage("test@" + DOMAIN, USER_ADDRESS);
+        Thread.sleep(5000);
```

Comment:
you don't want to assert on mail reception?

Reply:
The assertion is in the mpt test ValidateDeploymentWithMailsFromSmtp.test

Reply:
I mean you cannot await instead of sleeping?

Reply:
No, the send message is already  a blocking call. 
But i have to wait for the message to be dequeued before checking for its existence with IMAP

Reply:
so: why do you need to wait for it via mpt? you cannot wait here? and eventually check here for the result?